src/Commands/LightningCommands.php

```diff
@@ -54,9 +54,8 @@ protected function execute(InputInterface $input, OutputInterface $output) {
     if (!isset($info)) {
       throw new \Exception('Lightning info file not found.');
     }
-
-    $io = new DrupalStyle($input, $output);
-    $io->info($this->toSemanticVersion($info['version']));
+    $io = new DrushStyle($this->input(), $this->output());
```

Comment:
This might not be necessary. I think we might be able to just do `$this->output()->writeln('Version ' . static::toSemanticVersion(...))`.